linode_api4/objects/serializable.py

```diff
@@ -25,6 +25,8 @@ def from_json(cls, json: Dict[str, Any]) -> "JSONObject":
         """
         Creates an instance of this class from a JSON dict.
         """
+        if json is None:
+            return
```

Comment:
Could the return type be updated to suggest `None` is a valid return value?

e.g.
```python
def from_json(cls, json: Dict[str, Any]) -> Optional["JSONObject"]:
```